calculator-app/src/App.css

```diff
@@ -12,21 +21,61 @@
     animation: App-logo-spin infinite 20s linear;
   }
 }
+p.solid {border-style: solid;}
 
 .App-header {
   background-color: #282c34;
-  min-height: 100vh;
+  min-height: 10vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
```

Comment:
what is difference between align-items and justify content?
Is it necessary to use both?